db/migrate/20231201192630_create_posts.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+    def change
+        create_table :posts do |t|
+            t.integer :author_id
+            t.string :title
+            t.text :text
+            t.integer :comments_counter, default: 0
+            t.integer :posts_counter, default: 0
```

Comment:
Kindly note that the `posts_counter` attribute should exist in the `User` table as illustrated in the [ERD diagram](urlhttps://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png)

![image](https://github.com/bhushan354/Rails-Blog-App/assets/38088818/8fc0f0be-3f8e-4afd-9584-a8e377ae0f88)

- Feel free to modify the migration files directly by cutting line 8 and paste it into the Create User migration file. You have shown knowledge in generating tables in the database; there's no need to start afresh.
> You might need to drop the database `rails db:drop` and run migrations again `rails db:setup` to update your local database with the changes.